public/pages/Dashboard/components/FindingsDashboard/FindingFlyout.js

```diff
@@ -109,14 +107,10 @@ export default class FindingFlyout extends Component {
       >
         <EuiFlyoutHeader hasBorder>
           <EuiFlexGroup justifyContent="flexStart" alignItems="center">
-            <EuiFlexItem>
-              <EuiFlexGroup alignItems="center">
-                <EuiFlexItem>
-                  <EuiTitle size={'m'}>
-                    <h3 id={findingId || `temp_finding_${docId}`}>Document finding</h3>
-                  </EuiTitle>
-                </EuiFlexItem>
-              </EuiFlexGroup>
+            <EuiFlexItem className="eui-xScrollWithShadows">
+              <EuiTitle size={'m'}>
```

Comment:
Why did we add this class `eui-xScrollWithShadows`?

Reply:
It was one of reasons for cypress failing. Example alert/finding name was very long single word so X button was pop out of viewport. `eui-xScrollWithShadows` actually adds x-overflow on that flex item with kind of gradient for effect.

https://eui.elastic.co/v34.6.0/#/utilities/css-utility-classes#overflows

Reply:
How does it look like with the scroll bar? Can you add a screenshot and tag Ming for confirmation?

Reply:
moved to `eui-textTruncate`